src/bin/nydusd/fs_cache.rs

```diff
@@ -104,10 +110,10 @@ impl TryFrom<&[u8]> for FsCacheMsgHeader {
 /// from the fscache driver.
 #[derive(Default, Debug, Eq, PartialEq)]
 struct FsCacheMsgOpen {
-    fd: u32,
-    flags: u32,
     volume_key: String,
     cookie_key: String,
+    fd: u32,
+    flags: u32,
```

Comment:
Seems this change is unnecessary, have encountered any issue here?

Reply:
My own assumption is that is more friendly to CPU cache.